File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/Keys.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class Keys {
    public static final String F1 = "f1";
    public static final String F2 = "f2";
    public static final String F3 = "f3";
    public static final String F4 = "f4";


    @SerializedName(F1)
    private String f1;

    @SerializedName(F2)
    private String f2;

    @SerializedName(F3)
    private String f3;

    @SerializedName(F4)
    private String f4;

    public String getF1() {
        return f1;
    }

    public String getF2() {
        return f2;
    }

    public String getF3() {
        return f3;
    }

    public String getF4() {
        return f4;
    }

    public void setF1(String f1) {
        this.f1 = f1;
    }

    public void setF2(String f2) {
        this.f2 = f2;
    }

    public void setF3(String f3) {
        this.f3 = f3;
    }

    public void setF4(String f4) {
        this.f4 = f4;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/Keys.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class Keys {
    @SerializedName(F1)
    var f1: String? = null

    @SerializedName(F2)
    var f2: String? = null

    @SerializedName(F3)
    var f3: String? = null

    @SerializedName(F4)
    var f4: String? = null

    fun f1(): String? {
        return f1
    }

    val getF2: String?
        get() = f2

//    fun getF2(): String? {
//        return f2
//    } this.f3 = f3

    val getF3: String?
        get() = f3

    val getF4: String?
        get() = f4

    fun f1(f1: String?) {
        this.f1 = f1
    }

    fun f2(f2: String?) {
        this.f2 = f2
    }

    fun f3(f3: String?) {
        this.f3 = f3
    }

    fun f4(f4: String?) {
        this.f4 = f4
    }

    companion object {
        const val F1 = "f1"
        const val F2 = "f2"
        const val F3 = "f3"
        const val F4 = "f4"
    }
} //end of class
